tests/Doctrine/Tests/DBAL/Functional/Schema/SqliteSchemaManagerTest.php

```diff
@@ -294,4 +294,40 @@ public function testOnlyOwnCommentIsParsed() : void
             ->getColumn('col1')
             ->getComment());
     }
+
+    public function testCreatingATableWithAForeignKey() : void
+    {
```

Comment:
Few tests here have a guard clause about `sqlite3`, should I add one?

Reply:
Is there anything specific to SQLite in this scenario? If yes, then I think the setup of the functional platform-specific test should require the corresponding platform to be active.

Otherwise, the test should be part of a generic suite.

Reply:
I think this is platform independent, and I'm not sure what a platform-independent suite is. I moved it to a file that inherits from `DbalFunctionTestCase`, let's see how this pans out. My only doubt is about the foreign key name, I have no idea what computes it and if it will be the same for every platform.